zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/config/OpenApiResourceConfig.java

```diff
@@ -27,4 +41,53 @@ public void addResourceHandlers(final ResourceHandlerRegistry registry) {
         .resourceChain(true)
         .addTransformer(new YamlToJsonResourceTransformer(objectMapper));
   }
+
+  @Bean
+  public GroupedOpenApi restApiV2() {
+    return GroupedOpenApi.builder()
+        .group("Orchestration Cluster API")
+        .addOpenApiCustomizer(this::customizeOpenApi)
+        .pathsToMatch("/v2/**")
+        .build();
+  }
+
+  private void customizeOpenApi(final OpenAPI openApi) {
```

Comment:
💭 I assume this parsing only happens once when the bean is associated and not on every request?